.github/workflows/golang.yml

```diff
@@ -48,13 +45,41 @@ jobs:
         version: latest
         args: -v --timeout 5m
         skip-cache: true
+    - name: Setup JFrog CLI
+      id: jfrog
+      uses: jfrog/setup-jfrog-cli@v4
+      env:
+        JF_URL: https://${{ secrets.OIDC_ARTIFACTORY_ENDPOINT }}/
+        OIDC_USER: ${{ steps.jfrog.outputs.oidc-user }}
```

Comment:
Remove this `steps.jfrog.outputs.oidc-user` will only be available after jfrog runs